tests/unit/test_models.py

```diff
@@ -73,7 +71,7 @@ def __init__(self, *args, **kwargs):
         data = Column(String)
     engine.bind(Blob)
 
-    assert Blob.Meta.init is Blob
+    assert isinstance(Blob.Meta.init(), Blob)
```

Comment:
The docstring of this test (`test_load_default_init`) needs to be changed; the default model loader should *not* be `__init__` anymore.

Reply:
👍